fixtures/library-file/src/library.js

```diff
@@ -0,0 +1,12 @@
+import styles from './styles.less';
+import { customBox } from './styles.treat';
```

Comment:
We should probably stop adding fixtures with LESS and treat, to make it easier to migrate the future.

Reply:
Agreed to keep this for now and look into updating the fixtures to be more concise/fit for purpose across the board at a later date.